### ISSUE TITLE
Feature Request: Mute button when timer finishes

### ISSUE THREAD

**Issue by BrianKmdy** [Dziemborowicz/Hourglass#233]
I think it would be great if there was an ability to mute the timer after it finishes while still maintaining the red/flashing expired state. I like to have looping sound on my alarms but I can't always stop what I'm doing immediately. It would be nice to be able to mute the alarm but still keep it in its expired state.

**Comment by dziemborowicz** [comment]
That's an interesting request. You can temporarily mute the Hourglass application using Windows's built-in sound mixer (see, e.g., [here](https://www.askdavetaylor.com/adjust-volume-mute-per-app-basis-windows-11-pc/)), though this isn't exactly as convenient as a built-in function would be.

**Comment by BrianKmdy** [comment]
Agreed that that's not quite as convenient - that takes a few clicks through menus, whereas I have the timer set to always stay on top so it's right there and easy to click.
